resources/model/OpenStudio.idd

```diff
@@ -5888,7 +5888,6 @@ OS:ShadingControl,
        \note When Group is used the entire list is controlled simultaneously and if glare control is needed the entire
        \note group of window shades are deployed together a the same time
   A13; \field Sub Surface Name
-       \required-field
```

Comment:
@jmarrec I'm thinking this is because we don't have `\extensible:1` at the header of this object, not because of `\required-field`. I'm going to add `\required-field` back in, and then bump `\min-fields 4` up to 15 (i.e., shading control has to be assigned to at least one subsurface).

Reply:
:+1: